splinterd/src/config/scabbard_storage.rs

```diff
@@ -12,8 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::fmt::Display;
+
 #[derive(Debug, Copy, Clone)]
 pub enum ScabbardStorage {
     Database,
     Lmdb,
 }
+
+pub enum ScabbardStorageError {
+    ParseError(String),
+}
+
+impl Display for ScabbardStorageError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            ScabbardStorageError::ParseError(msg) => write!(f, "--scabbard-storage={}", msg),
```

Comment:
This error should not mention the CLI argument as it could occur in other cases such as loading a toml file with an incorrect value.

Reply:
Im happy to change this but it should be noted its currently dead code, invalid args get caught by serdes error handling coming from the toml and by claps coming from the cli. If clap didn't have the option to provide possible values this code could be hit.

Reply:
Yeah